setup.cfg

```diff
@@ -136,6 +136,7 @@ filterwarnings =
     # Temporarily ignore warnings internal to Python 3.9.7, can be removed again in 3.9.8.
     ignore:The loop argument is deprecated since Python 3.8, and scheduled for removal in Python 3.10.:DeprecationWarning:asyncio
     ignore:Creating a LegacyVersion has been deprecated and will be removed in the next major release:DeprecationWarning::
+    ignore::ResourceWarning
```

Comment:
This lacks a comment on why this is necessary and under what conditions it can be removed.

Reply:
This has now been removed.